cromwell_tools/utilities.py

```diff
@@ -232,3 +238,95 @@ def validate_cromwell_label(label_object):
 
     if err_msg != '':
         raise ValueError(err_msg)
+
+
+def prepare_workflow_manifest(wdl_file, inputs_files=None, options_file=None, dependencies=None, label_file=None,
+                              collection_name=None, on_hold=False):
+    """Prepare the submission manifest for a workflow submission.
+
+    Args:
+        wdl_file (Union[str, io.BytesIO]): The workflow source file to submit for execution. Could be either the path
+            to the file (str) or the file content in io.BytesIO.
+        inputs_files (Optional[Union[List[Union[str, io.BytesIO]], str, io.BytesIO]]): The input data in JSON
+            format. Could be either the path to the file (str) or the file content in io.BytesIO. This could also
+            be a list of unlimited input file paths/contents, each of them should have a type of
+            Union[str, io.BytesIO]. (default None)
+        options_file (Optional[Union[str, io.BytesIO]]): The Cromwell options file for workflows. Could be either
+        the path to the file (str) or the file content in io.BytesIO. (default None)
+        dependencies (Optional[Union[str, List[str], io.BytesIO]]): Workflow dependency files. Could be the path to
+            the zipped file (str) containing dependencies, a list of paths(List[str]) to all dependency files to be
+            zipped or a zipped file in io.BytesIO. (default None)
+        label_file(Optional[Union[str, _io.BytesIO]]): JSON file containing a collection of key/value pairs for workflow
```

Comment:
Is there a particular reason why this now only allows adding a file containing labels instead of allowing users to pass them in directly?

Reply:
No except that I somehow wanted to keep the signature closer to the Cromwell endpoint... I think what we can do is let the user pass them in directly but do some magic conversion with the CLI (e.g. `io.BytesIO(json.dumps(kwargs))`), that way we don't make this method too complicated, what do you think?